src/registries/canister_registry.ts

```diff
@@ -0,0 +1,73 @@
+import { HttpAgent, ActorSubclass } from '@dfinity/agent';
+import fetch from 'cross-fetch';
+
+import CanisterRegistryInterface from '../interfaces/dab_registries/canister_registry';
+import IDL from '../idls/dab_registries/canister_registry.did';
+import { IC_HOST } from '../constants';
+import Registry from './standard_registry';
+import { generateActor } from '../utils/actorFactory';
+import { formatMetadata, FormattedMetadata } from '../utils/registry';
+import { Principal } from '@dfinity/principal';
+
+const CANISTER_ID = 'curr3-vaaaa-aaaah-abbdq-cai';
+const DEFAULT_AGENT = new HttpAgent({ fetch, host: IC_HOST });
+
+interface CanisterMetadata {
```

Comment:
Is this for the old version?

Reply:
yes, this is for backwards compatibility on old functions!